lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=25
+SIDEKIQ_DB_POOL_SIZE=26
```

Comment:
Wooooo that's a high thread count. The assumption you're making is that the average job is less than 4% CPU or more than 96% I/O. Definitely not the norm except for apps that mostly use jobs to call into APIs...

A default of 5 threads would make much more sense to me.